layer/filestore_unix.go

```diff
@@ -0,0 +1,13 @@
+// +build !windows
```

Comment:
Not for now 😅 
I just meant that the interface could be used for future extension